lib/system/mmdisp.nim

```diff
@@ -82,8 +82,9 @@ else:
     # XXX due to bootstrapping reasons, we cannot use  compileOption("gc", "stack") here
     include "system/gc_regions"
   elif defined(nimV2) or usesDestructors:
-    var allocator {.rtlThreadVar.}: MemRegion
-    instantiateForRegion(allocator)
+    when not defined(useNimRtl):
+      var allocator {.rtlThreadVar.}: MemRegion
+      instantiateForRegion(allocator)
```

Comment:
Refc uses this condition too => https://github.com/nim-lang/Nim/blob/27e548140b0f51a8dbc6f6094764e02f08ae509e/lib/system/gc.nim#L149